mmdeploy/backend/sdk/export_info.py

```diff
@@ -159,6 +159,18 @@ def get_preprocess(deploy_cfg: mmengine.Config, model_cfg: mmengine.Config,
     task_processor = build_task_processor(
         model_cfg=model_cfg, deploy_cfg=deploy_cfg, device=device)
     transforms = task_processor.get_preprocess()
+
+    # remove scope name
+    # NOTE: There is a potential risk that transforms with same names in
+    # different scope might have different behavior
+    for transform in transforms:
+        trans_type = transform['type']
+        trans_type = trans_type.split('.')
+        if len(trans_type) > 1:
+            trans_type = trans_type[1:]
+        trans_type = '.'.join(trans_type)
+        transform['type'] = trans_type
```

Comment:
If the part only works for MMRotate, we may consider moving it to `get_preprocess` of mmrotate.

Reply:
I am not sure. This would happen when a repo reuse modules/transforms in another repo.

Reply:
By far, only mmrotate shows the attribute. Maybe MMOCR in the future.